### ISSUE TITLE
How to set indentation for mobile version if grid text label has large value with suffix?

### ISSUE THREAD

**Issue by adminoid** [IonDen/ion.rangeSlider#750]
Please see the screen:
<img width="390" alt="image" src="https://user-images.githubusercontent.com/548475/114339587-41e9bb00-9b5e-11eb-92bc-20c843d02db9.png">

I need to make label grid points in a mobile mode less frequently for normal display (to avoid interlaps).

**Comment by adminoid** [comment]
And one more thing...
`ionRangeSlider({values: scale(), ...})`
where scale - function who returns array.

Thanks in advance!

**Comment by IonDen** [comment]
Yes, this is a bug on a small screens with too dense grid. Unfortunately all you can do:
1. Provide alternative `values` array with less items.
2. Use normal slider without values array. Use `step` instead and then you will be able to use `grid_num` param.

**Comment by adminoid** [comment]
Thank you very much!
And another way - for small screen make `grid: false,` and in other cases `grid: true,`.
But running ionRangeSlider constructor many times can be a load on the browser if screen width changes dynamically.

**Comment by IonDen** [comment]
With a `values` array the grid always shows one label per entry, so on a narrow screen with long labels they crowd into each other. Two ways out: give the slider a shorter `values` array on small screens, or drop the array entirely and use a normal numeric slider with `step`, then thin the labels out with `grid_num`.

```js
$(".js-range-slider").ionRangeSlider({
    min: 0,
    max: 100,
    step: 1,
    grid: true,
    grid_num: 5
});
```

`grid_num` controls how many big ticks get drawn regardless of screen width, so pick a lower number for mobile.

Drop the grid_num lower if mobile spacing is still tight.
